plugins/ngrok/provisioner.go

```diff
@@ -42,14 +42,22 @@ func ngrokProvisioner() sdk.Provisioner {
 	// semver.Compare resulting in 0 means 3.2.1 is in use
 	// semver.Compare resulting in +1 means >3.2.1 is in use
 	if semver.Compare(currentVersion, envVarAuthVersion) >= 0 {
-		return ngrokEnvVarProvisioner{}
+		return true
 	}
 
 	// Otherwise use config file to provision credentials
-	return fileProvisioner{}
+	return false
 }
 
 func (f fileProvisioner) Provision(ctx context.Context, in sdk.ProvisionInput, out *sdk.ProvisionOutput) {
+	ngrokProvisioner := IsNgrokAPIKeySupported()
+
+	if ngrokProvisioner {
```

Comment:
nit - we can call this function directly in the if statement, to avoid initializing a variable that's only used once.

Reply:
Done never knew you could do this.. thanks!